### PR TITLE
Fix new site command in bootstrap documentation

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ Then edit `scripts/mass_patch.config.sh` and add the proper project names.
 Then, you can create a new site in Pantheon which can also be done with a
 [terminus command](https://pantheon.io/docs/guides/drupal8-commandline):
 
-    ddev exec terminus site:create my-site "My Site" "Drupal 9"
+    ddev exec terminus site:create my-site "My Site" "Drupal 10 Start State"
```
Comment (amitaibu) [inline README.md:257]: What do you mean by "Start State"?

Comment (AronNovak) [reply README.md:257]: This is how Pantheon calls it.
See:
```
 aaron   main  ~  gizra  drupal-starter  terminus  upstream:list
 ------------------------ ----------------------- ------------------------ -------- ----------- ---------------- 
  ID                       Name                    Machine Name             Type     Framework   Organization    
 ------------------------ ----------------------- ------------------------ -------- ----------- ---------------- 
  19a917a4-c56e-44d6-98a   Decoupled Drupal 10     decoupled-drupal-10-co   core     drupal8                     
  1-49e7f779e678           Composer Managed        mposer-managed                                                
  c9f5e5c0-248f-4205-b63   Decoupled WordPress     decoupled-wordpress-co   core     wordpress                   
  a-d2729572dd1f           Composer Managed        mposer-managed                                                
  bde48795-b16d-443f-af0   Drupal (Composer        drupal-composer-manage   core     drupal8                     
  1-8b1790caa1af           Managed)                d                                                             
  7f9fd7d4-77ec-4e6b-a73   Drupal 10 Start State   drupal-10-composer-man   core     drupal8                     
  f-d32b93a1c6b1                                   aged                                                          
  21e1fada-199c-492b-97b   Drupal 7                drupal7                  core     drupal                      
  d-0b36b53a9da0                                                                                                 
  8a129104-9d37-4082-aaf   Drupal 8                drupal8                  core     drupal8                     
  8-e6f31154644e                                                                                                 
  e8fe8550-1ab9-4964-883   WordPress               wordpress                core     wordpress                   
  8-2b9abdccf4bf                                                                                                 
  90a683cd-4e03-4832-9b4   WordPress (Composer     wordpress-composer-man   core     wordpress                   
  9-be97ab2a0be4           Managed)                aged                                                          
 ------------------------ ----------------------- ------------------------ -------- ----------- ---------------- 

```

Comment (AronNovak) [reply README.md:257]: For the record, in the bootstrap script, we refer if by `ID`.

Comment (AronNovak) [reply README.md:257]:
```
 aaron  ~  terminus site:create my-site "My Site" "Drupal 10"
 [error]  Could not find an upstream identified by Drupal 10. 
 aaron  ~  1  

```

Nah, it requires a 1-1 match for the upstream human-readable name.

**Comment by amitaibu** [comment]
Thanks

**Pull request merged**